app/assets/stylesheets/application.css

```diff
@@ -0,0 +1,173 @@
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family:
+    'Lucida Sans',
+    'Lucida Sans Regular',
+    'Lucida Grande',
+    'Lucida Sans Unicode',
+    Geneva,
+    Verdana,
+    sans-serif;
+}
+
+h1,
+h2,
+h3 {
+  color: #4a7856;
+  font-size: 2rem;
+}
+
+a {
+  text-decoration: none;
+  color: #fff;
+}
+
```

Comment:
your recipes and public recipes lists are not as wide as they suppose to look 
![image](https://user-images.githubusercontent.com/87448628/197344262-3fc3391a-8300-43c3-b054-1e2495a63e59.png)
Kindly Try to make the width fixed so it matches the given wireframe